api/package.json

```diff
@@ -5,14 +5,18 @@
   "main": "main.js",
   "scripts": {
     "audit": "npx audit-ci --config ./audit-ci.json",
+    "knex-console": "NODE_ENV=development node --experimental-repl-await knexrepl.js",
     "lint": "eslint '**/*.js'",
     "make-migrate": "knex migrate:make",
     "migrate": "knex migrate:latest",
+    "migrate-dev": "NODE_ENV=development DEV_DB_HOST=localhost npm run migrate",
+    "rollback": "knex migrate:rollback",
     "seed": "knex seed:run",
+    "seed-dev": "NODE_ENV=development DEV_DB_HOST=localhost npm run seed",
     "start": "node main.js",
-    "start-dev": "nodemon ./main.js -e js",
+    "start-dev": "NODE_ENV=development DEV_DB_HOST=localhost nodemon ./main.js -e js",
```

Comment:
Added `migrate-dev` and `seed-dev` commands that call `migrate` and `seed` commands with the appropriate env vars when developing locally.